experimental/ratify/proto/v1/common.proto

```diff
@@ -0,0 +1,33 @@
+syntax="proto3";
+
+package common;
+
+option go_package = "github.com/deislabs/ratify/experimental/proto/v1/common";
+
+message ArtifactDescriptor {
+    message Attributes {
+        map<string,string> values = 1;
+    }
+    string path = 1;
+    repeated Attributes attributes = 2;
+}
+
+message ArtifactReference {
+    string rawValue = 1;
+    ArtifactDescriptor descriptor = 2;
+}
+
+message Artifact {
```

Comment:
I'm a bit confused on the naming convention. According to the original design proposal, we would use Artifact as being “one conceptual piece of content stored as Blobs” however I think in this particular definition "Artifact" is referring to the specific OCI/ORAS Artifact which contains the specific`artifactType` field . Do we want to rename this to something like "Referrer"? Or can we drop the `Artifact` prefix for `ArtifactDescriptor` and `ArtifactReference`?